### ISSUE TITLE
NTR: Esophagus wide by barium swallow, CT, MRI, or endoscopy (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8967]
New term request Esophagus wide by barium swallow, CT, MRI, or endoscopy (hausma_120913192203): Esophageal; includes megaesophagus; widened  UMLS: 2073712

**Comment by pnrobinson** [comment]
@MickeySegal
what is the disease context?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000589
- label: Esophageal dilation
- parents: [Abnormal esophagus morphology (HP:0002031)](https://hpo.jax.org/app/browse/term/HP:0002031)
- pmid: PMID:26639310
- def: Widening (relatively uniform increase in diameter) of a segment of the esophagus.
- comment: This feature can be ascertained by chest computed tomography.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8967
- synonyms:
